scripts/validate-security.sh

```
@@ -0,0 +1,176 @@
+else
+  echo -e "${YELLOW}⚠️  .envファイルが見つかりません（GitHub Actions環境では正常）${NC}"
+  ((WARNINGS++))
+fi
+echo ""
+
+# 6. ハードコードされた機密情報のチェック
+echo "📋 6. ハードコードされた機密情報のチェック"
+HARDCODED_SECRETS=0
+
+# パスワードのハードコード検索
+if grep -rn "password.*=.*['\"].*['\"]" src/ --include="*.js" | grep -v "password.*=.*process.env" | grep -v "password.*=.*'***'" | grep -v "// " | wc -l | tr -d ' ' | grep -q "^0$"; then
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)